frontend/src/lib/api/access-list.ts

```diff
@@ -111,7 +112,7 @@ class AccessListService {
     identifier: string,
     listType: string,
     rules: Array<{ old_number: number; new_number: number; rule_data: AccessListRule }>
-  ): Promise<any> {
+  ): Promise<VyOSResponse> {
     const result = await apiClient.post("/vyos/access-list/reorder", {
```

Comment:
If you promise `<VyOSResponse>` you also have to update the `apiClient.post
```suggestion
    const result = await apiClient.post<VyOSResponse>("/vyos/access-list/reorder", {
```

Reply:
Sorry, my mistake, I missed that.